src/estop.cpp

```diff
@@ -56,11 +85,22 @@ int main(int argc, char **argv)
 
     initializeGPIO();
 
+    ros::XMLRPCManager::instance()->unbind("shutdown");
+    ros::XMLRPCManager::instance()->bind("shutdown", shutdownCallback);
```

Comment:
Shouldn't we use the same code to handle sigint and ros initiated shutdowns?

Reply:
It has been awhile, but I think that was the point of the PR: sigint and rosnode kill can't be handled in the same way and need their own code.

Reply:
yeah but I was talking about having the shutdown behaviour integrated in a private function so that it can affect both shutdown with one modification, let's say in the future, we want to turn off the lights on shutdown, that you'd have to add it to both shutdown routines

Reply:
Wouldn't be possible to add whatever that we want to shut down to the shutdownCallback function ?

Reply:
I think I get what you mean, but wouldn't that require a new function that would be called by both methods? Since sig and xml require different parameters in their shutdown functions.

Reply:
> I think I get what you mean, but wouldn't that require a new function that would be called by both methods? Since sig and xml require different parameters in their shutdown functions.

yeah that's what I meant but it's really overkill to isolate 1 line of code

Reply:
This is good to merge then?

Reply:
yup